delete this patch after openxla pin updated to date after or equal to 08/09
diff --git a/xla/service/layout_assignment.cc b/xla/service/layout_assignment.cc
index f69224f42..43feda2a0 100644
--- a/xla/service/layout_assignment.cc
+++ b/xla/service/layout_assignment.cc
@@ -154,7 +154,7 @@ OperandLayoutConstraint::OperandLayoutConstraint(
       instruction_(instruction),
       operand_no_(operand_no) {
   CHECK(shape_layout.LayoutIsSet());
-  CHECK(ShapeUtil::CompatibleIgnoringElementType(
+  CHECK(ShapeUtil::CompatibleKind(
       shape_layout.shape(), instruction->operand(operand_no)->shape()))
       << shape_layout.shape() << " is not compatible with "
       << instruction->operand(operand_no)->shape() << " (for operand "
